libraries/AP_HAL_ChibiOS/hwdef/FlywooF405HD-AIOv2/defaults.parm

```diff
@@ -0,0 +1,9 @@
+# setup for LEDs on chan9
+SERVO5_FUNCTION 120
+NTF_LED_LEN 2
+# ESC setup
+MOT_PWM_TYPE 5
+SERVO_BLH_AUTO 1
+SERVO_BLH_TRATE 0
+# RC setup
+RC_OPTIONS 8704
```

Comment:
sorry somehow these changes (which match the readme changes) that make it match the wiring diagram got lost during my review
```suggestion
RC_OPTIONS 8704
OSD_TYPE 5
SERIAL3_PROTOCOL -1
SERIAL2_PROTOCOL 2
SERIAL6_PROTOCOL 42
```